spec/policy/jwt_claim_check/jwt_claim_check_spec.lua

```diff
@@ -0,0 +1,127 @@
+local JWTClaimCheckPolicy = require('apicast.policy.jwt_claim_check')
+local ngx_variable = require('apicast.policy.ngx_variable')
+
+
+describe('JWT claim check policy', function()
+
+  local context = {
+    jwt = {
+      foo = "foo",
+      bar = "bar",
+      one = "1",
+      hundred= "100",
+      ooo = { 1,2},
+      roles = { "one", "two"}
+
+    }
+  }
+
+  before_each(function()
+    ngx.header = {}
+    stub(ngx, 'print')
+
+    stub(ngx.req, 'get_method', function() return 'GET' end)
+    -- avoid stubbing all the ngx.var.* and ngx.req.* in the available context
+    stub(ngx_variable, 'available_context', function(ctx) return ctx end)
+  end)
+
+  describe("plain template match_type", function()
+    it("valid claim", function()
+      local jwt_check = JWTClaimCheckPolicy.new({operations={
+          {op="==", match="foo", match_type="plain", value="foo"},
+        }})
+      jwt_check:access(context)
+      assert.not_same(ngx.status, 403)
+    end)
+
+    it("invalid claim", function()
+      local jwt_check = JWTClaimCheckPolicy.new({operations={
+          {op="==", match="invalidClaim", match_type="plain", value="foo"},
+        }})
+      jwt_check:access(context)
+      assert.same(ngx.status, 403)
+    end)
+
+  end)
+
+  describe("Text operations", function()
+    it("equal operation", function()
+      local jwt_check = JWTClaimCheckPolicy.new({operations={
+          {op="==", match="{{foo}}", match_type="liquid", value="foo"},
```

Comment:
I think this is a bit confusing. The name of the claim and its value are the same, so it's not immediately obvious what is being tested.